web/themes/custom/pineapple/templates/field/field--field-bp-body.html.twig

```diff
@@ -38,5 +38,5 @@
  */
 #}
 {% include "@components/_field.twig" with {
-  attributes: create_attribute().addClass('u-o-3')
```

Comment:
This change now means there's a spacing inconsistency between the basic pages and the recipes/articles. Is this deliberate? An argument can be made for either case

Reply:
It is deliberate actually. The spacing increase is not necessary on the article and recipe content types which tend to have a different structure and are heavily text based with any additional components just adding to the main content of the page. Basic pages are more varied and the components are more likely to be unrelated and require more spacing between them

Reply:
Okay No problem.